master/getting-started/mesos/tutorials/connecting-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Connecting to Tasks
+---
+
+Calico assigns each mesos task a uniquely addressable IP address.
+In testing, it is easy to test connectivity by making requests directly to this
+IP.
+
+However, when designing a multi-tier application, it is not usually sensible to
```

Comment:
Is that really an issue with multi-tier.  More an issue of short task lifetimes.